src/core/tasks_pool.h

```diff
@@ -0,0 +1,26 @@
+#include <malloc.h>
+#include "xccl_global_opts.h"
+#include "xccl_schedule.h"
+
+
+#define LINE_SIZE 256
+
+extern xccl_config_t xccl_lib_global_config;
+
+typedef struct context {
+    int increase_pool_locks[2];
+    int linked_list_locks[2];
+    ucc_coll_task_t ***tasks;
+    unsigned int which_pool;
+    ucc_coll_task_t *linked_lists[2];
+    int deque_size;
+} context;
```

Comment:
1. alignment
2. context is not  a good name, rename it to xccl_task_pool_t or something like that

Reply:
fixed